### PR TITLE
tmuxinator 1.1.1 (new formula)

### PR DESCRIPTION
**Pull request by nexeck**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/tmuxinator.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
+class Tmuxinator < Formula
+  desc "Manage complex tmux sessions easily"
+  homepage "https://github.com/tmuxinator/tmuxinator"
+  url "https://github.com/tmuxinator/tmuxinator/archive/v1.1.1.tar.gz"
+  sha256 "cc293578bca43ba5cf0d60c1355c6aa1da9d923a0acc274a47ceab03812a6ef4"
+  head "https://github.com/tmuxinator/tmuxinator.git"
+
+  depends_on "ruby" => "2.4.6"
```
Comment (SMillerDev) [inline Formula/tmuxinator.rb]: This is not valid syntax. And as the bot pointed out, isn't this version available for all macOS targets?

Comment (nexeck) [reply Formula/tmuxinator.rb]: I removed the version requirement.
If i remove the hole depends_on "ruby" line, i get this error:

```
ERROR:  Error installing tmuxinator-1.1.1.gem:
tmuxinator requires Ruby version >= 2.4.6.
```

Comment (nexeck) [reply Formula/tmuxinator.rb]: @SMillerDev can you explain me, what is wrong with this syntax?
I used this documentation for the depends_on part: https://github.com/Homebrew/brew/blob/master/docs/Formula-Cookbook.md#specifying-other-formulae-as-dependencies

Comment (SMillerDev) [reply Formula/tmuxinator.rb]: You're mixing `:requirement => "version"` with `"requirement" => :stage` which are very different things.

Comment (nexeck) [reply Formula/tmuxinator.rb]: Thank you for explaining this. Now i understand the difference :-)

Is my change good, which makes the depends_on line valid?
I do not know, why i get the ruby version error if i skip this line.

**Comment by nexeck** [comment]
If something else is missing, just let me know :-)

**Comment by SMillerDev** [comment]
Thanks @nexeck! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Comment by kenden** [comment]
Tested with 
```bash
$ brew tap nexeck/homebrew-tmuxinator
==> Tapping nexeck/tmuxinator
Cloning into '/usr/local/Homebrew/Library/Taps/nexeck/homebrew-tmuxinator'...
remote: Enumerating objects: 5, done.
remote: Counting objects: 100% (5/5), done.
remote: Compressing objects: 100% (4/4), done.
remote: Total 5 (delta 0), reused 4 (delta 0), pack-reused 0
Unpacking objects: 100% (5/5), done.
Tapped 1 formula (30 files, 25.3KB).

$ brew install nexeck/homebrew-tmuxinator/tmuxinator
Updating Homebrew...
==> Installing tmuxinator from nexeck/tmuxinator
==> Downloading https://github.com/tmuxinator/tmuxinator/archive/v1.1.1.tar.gz
==> Downloading from https://codeload.github.com/tmuxinator/tmuxinator/tar.gz/v1.1.1
######################################################################## 100.0%
==> gem build tmuxinator.gemspec
==> gem install tmuxinator-1.1.1.gem
🍺  /usr/local/Cellar/tmuxinator/1.1.1: 1,388 files, 2.4MB, built in 14 seconds

$ tmuxinator
tmuxinator commands:
  tmuxinator commands                          # Lists commands available in tmuxinator
(...)
```

**Comment by chenrui333** [comment]
Thanks for your contribution!

**Pull request closed without merging**